ding2.install

```diff
@@ -624,3 +624,28 @@ function ding2_update_7051() {
   // Opensearch.
   ding2_update_7048();
 }
+
+/**
+ * Update cookie compliance body text and removes Webtrekk/Webtrends specific
+ * part, which is now handled in ding_webtrekk module.
+ */
+function ding2_update_7052() {
+  if ($path = drupal_lookup_path('source', 'cookies')) {
+    $path = explode('/', $path);
+    $node = node_load($path[1]);
+    $node->field_ding_page_body = array(
+      'und' => array(
+        array(
+          'value' => _ding2_cookie_compliance_text(),
+          'format' => 'ding_wysiwyg',
+          'safe_value' => _ding2_cookie_compliance_text(),
+        ),
+      ),
+    );
+    node_save($node);
```

Comment:
This will override any modifications to the text that editors might have made locally. I do not know if there is anything we can do about this.

Reply:
Maybe we could copy the old text to a new node?

Reply:
Update the PR to make backup of old cookie page on update.